### DISCUSSION TITLE
Welcome to pixi Discussions!

### DISCUSSION THREAD

**Discussion by ruben-arts** [prefix-dev/pixi#270] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by twrightsman** [comment]
Hello! I'm a graduate student experimenting with machine learning models for crop plant genomic data. I use Conda and `conda-libmamba-solver` to mange all of my GPU machine learning environments with minimal headache.

`pixi` excites me for many reasons:
1. No `(base)` environment to accidentally mess up
2. No complex installer, just copy a single binary to `$PATH`
3. Potentially easier to package upstream into Debian than base Conda
    - doing `sudo apt install pixi` would be a dream!
4. Defaults to `conda-forge` (I think?)

I typically tend to create a few environments for a given project in an `envs/` directory and activate among them as needed. I suppose I can adapt `pixi` to this use case by just running `pixi shell` inside the respective environment project directory instead of `conda activate`?

**Reply by ruben-arts** [reply]
He @twrightsman, I'm sorry for missing this message!

Really good to hear that you're already having a good experience with the conda ecosystem and that you're exited about pixi!

Your correct about all your reasons, the apt install might be a longer journey but never say never!

The `pixi shell`'s can be overlapped as you mention. This is however not how we designed pixi to be used, so we don't really support this behavior.

In the future we will possibly have the ability to define multiple environments in one `pixi.toml` maybe you would be interested in that!

**Reply by twrightsman** [reply]
All good @ruben-arts !

I have some familiarity with Debian packaging so when pixi stabilizes a bit I can help organize that effort, if needed.

Regarding `pixi shell`, do you mean activating nested environments? I don't have a need for that, but that seems potentially cool :) like an OSTree overlay. I would exit any active environment before activating another. I can see how nested envs would quickly become a pain to debug.

**Reply by ruben-arts** [reply]
That's good to know! 

Ah yeah I thought you where talking about nested once indeed.
